packages/cxl-ui/scss/global/cxl-adeft-accordion.scss

```diff
@@ -0,0 +1,55 @@
+@use "~@conversionxl/cxl-lumo-styles/scss/mq";
+@use "../mixins";
+
+cxl-adeft-accordion {
```

Comment:
"Adeft" product name is history.

All code should be refactored to `cxl-playbook-accordion` name.

Reply:
> "Adeft" product name is history.
> 
> All code should be refactored to `cxl-playbook-accordion` name.

"adeft" removed in last commit